test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -30,6 +30,8 @@
     # Internal
     ("static", datetime.date(9999, 1, 1)),
     ("prim::ModuleDictIndex", datetime.date(9999, 1, 1)),
+    ("aten::prim::MKLDNNRelu6", datetime.date(9999, 1, 1)),
+    ("aten::prim::MKLDNNRelu6_", datetime.date(9999, 1, 1)),
```

Comment:
looks like we need `prim::MKLDNNRelu6_` and `prim::MKLDNNRelu6` as well?

Reply:
Oh gee, what a typo! Thanks!